core/frontend/src/ViewCreator3d.ts

```diff
@@ -91,8 +91,15 @@ export class ViewCreator3d {
     const categories: Id64Array = await this._getAllCategories();
 
     // model extents
+    let modelExtents: Range3d = this._imodel.projectExtents;

```

Comment:
Don't initialize to project extents - that may make the range larger than we want.

Reply:
Pushed fixes.